app/tests/test_assets_model.py

```diff
@@ -0,0 +1,90 @@
+import logging
+
+from django.core.exceptions import ValidationError
+from django.test import TestCase
+
+from stac_api.models import Asset
+from stac_api.models import Collection
+
+from tests.data_factory import Factory
+from tests.utils import mock_s3_asset_file
+
+logger = logging.getLogger(__name__)
+
+
+class AssetsModelTestCase(TestCase):
+
+    @classmethod
+    @mock_s3_asset_file
+    def setUpTestData(cls):
+        cls.factory = Factory()
+        cls.collection = cls.factory.create_collection_sample(db_create=True).model
+        cls.item = cls.factory.create_item_sample(
+            collection=Collection.objects.get(name="collection-1"), db_create=True
```

Comment:
Here you can simply do `collection=cls.collection` which is safer because the name of the created collection is not guaranteed.

Reply:
Done